_data/Tools/ENR4.1parser.py

```diff
@@ -0,0 +1,55 @@
+import requests
+from bs4 import BeautifulSoup
+from Airac import Airac
+
+def capitalConvert(inp):
+    out = []
+    for word in inp.split(" "):
+        out.append(word[0].upper() + word[1:].lower())
+    
+    return " ".join(out)
+
+def convertCoords(inA, inB):
+    # convert from XXXXXX.XXN to NXXX.XX.XX.XXX etc
+    outA = inA[-1] + "0" + inA[:2] + "." + inA[2:4] + "." + inA[4:6] + "." + inA[7:9].ljust(3, '0')
+    outB = inB[-1] + inB[:3] + "." + inB[3:5] + "." + inB[5:7] + "." + inB[8:10].ljust(3, '0')
+    return [outA, outB]  # probably should be a tuple
+
+cycle = Airac()
+
+url = cycle.url() + "EG-ENR-4.1-en-GB.html"
+
+text = requests.get(url).text
+
+soup = BeautifulSoup(text, "html.parser")
+
+ad22 = soup.find("div", attrs={"id": "ENR-4.1"})
+rows = list(list(ad22.children)[1].children)[1].children
+
+with open("output.txt", "w") as f:
```

Comment:
Will this write to _data/Tools/output.txt - if so a more descriptive file name might be handy so if we add other tools they don't all overwrite each other? Alternatively could consider using sub-folders in Tools, not sure what you think?

Reply:
Yes, it'll all be different quite soon - I've got another PR coming with the tool rewrites. I could probably just remove the tool rewrites from this PR and #4823, and then update them all in one. What do you think?